### ISSUE TITLE
Distorted path

### ISSUE THREAD

**Issue by cdoublev** [colinmeinke/svg-arc-to-cubic-bezier#10]
Hello,

I have created the [following test case](https://gist.github.com/cdoublev/1912e3806e960fa36051a1a311396b08#file-index-html) to illustrate the drift that occurs in my path definition, using `svg-arc-to-cubic-bezier` to transform arcs to cubic curves.

```html
<script src="./svg-arc-to-cubic-bezier.js"></script>
<!-- <script src="./a2c.js"></script> -->
```

You can switch between this package and the one below, which is the function used in Snap.svg/SVGO. You will see that both paths are visually identical when using the latter. But it returns relative values and I needed absolute ones so I gave your function a shot.

I didn't change the functions signatures. I just added a utility to concat the returned cubic points into a string. I tested your function with a very simple arc and it works perfectly.

Are you aware of some corner cases/bugs/limitations with the maths involved in `svg-arc-to-cubic-bezier`? Would you be interested to get a feed back if I implement the function from Snap.svg/SVGO by returning absolute values?

**EDIT:** I implemented it [here](https://github.com/creativewave/use-definition/blob/master/src/definition/getCubicFromArc.js). But I'm still interested to check and learn about the maths involved here... someday.

**Comment by wcandillon** [comment]
I am also getting slight deformation via this function.

**Comment by colinmeinke** [comment]
@cdoublev @wcandillon sorry about that. I believe this issue is caused by https://github.com/colinmeinke/svg-arc-to-cubic-bezier/pull/4, which was released in 3.1.0. I've recreated https://codepen.io/colinmeinke/pen/NZLobO, and if you change the dependency to 3.0.0 it works as expected.

We found an issue with this already on basic arcs and fixed https://github.com/colinmeinke/svg-arc-to-cubic-bezier/issues/7, but looking at this more closely I think the PR needs to be reverted entirely as I'm pretty sure it's incorrect.
